File: LeetCode/Easy/0389-find-the-difference/0389-find-the-difference.c
char findTheDifference(char* s, char* t) {
    char res=0;
    int i=0,j=0;
    for(; s[i] && t[j]; i++,j++){
        res ^= s[i];
        res ^= t[j];
    }
    return res ^ t[j];
}

File: LeetCode/Easy/0389-find-the-difference/0389-find-the-difference.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        res  = 0

        for i in range(len(s)):
            res ^= ord(s[i]) ^ ord(t[i])

        res ^= ord(t[-1])
        
        return chr(res)
        